ssn/ssn_separated/ssn.ttl

```diff
@@ -890,47 +773,41 @@ ssn:SensorInput rdf:type owl:Class ;
                                   owl:allValuesFrom ssn:Property
                                 ] ;
                 dc:source "http://www.w3.org/2005/Incubator/ssn/" ;
-                rdfs:comment "An Event in the real world that 'triggers' the sensor.  The properties associated to the stimulus may be different to eventual observed property.  The event, not the object, triggers the sensor." ;
+                rdfs:comment "An event in the real world that 'triggers' the sensor. The properties associated to the stimulus may be different to the eventual observed property. The event, not the object, triggers the sensor." ;
```

Comment:
An event in the real world that 'triggers' the Sensor. The properties associated to the Stimulus may be different to the eventual observed ObservableProperty. The event, not the object, triggers the Sensor.

did we decide to keep Stimulus and SensorInput as aliases ? (they are equivalent)